tests/data/cover_crypt/non_regression/non_regression_test_vector.json

```diff
@@ -1,37 +1,37 @@
+  "public_key": "8nFFUE4ypftlUndhfXchQvpM20YpxVCU5vEYrXS+txNyQBE6AyS1mX4AEXL3SZ8EZehXH7W70ORgXTo9Th7/LxQCAwYA2vt1IZWcWVm2IcXBQ2PVjWT3YEM+NBiuey6WzaEKcnsCBQgAAFaxMalHDTMkEExrNkqIw+ZLLf7Dcyq+rVYAQ2SO9UICAgYAFmNUgemd3S3xJIaY9jkF5xiUbpOzGrfKOuWlM5PjYX0CAwkAhPAeLeOxPZAbO5sy29VXvmpng7YBzg0k7lfYbbgV6E8CBAcA8O+6ZQqYu/LeZv/TUo7VUhEK6g1osDrjjBQbidu5qGoCAggAANjVj65TpKRXBWGI5EtJitOzsd4KZe8X8l2X8yLsLAECAQkAkGsvAkZdloqJ6KHYiyEGcCUiF5ldKqw+iWE04ccWPigCBAgAJg0OSZD4a0E4Z9e88bBOlKoOj44QZTw2KoBW7W6Kpj4CBAkAHjkXre3eS08RtjJ25P/LdL5soEYsxJRuGGWA84qAmEACAQgAikJ9lAXl0IVZJ60zJJOWjemZfGzAVp7hCEgGu0jNy2QCBQcA6LAcNOCJqTuXBkdtMbFWkJPNDgnpKIPGSCEAlXXaEm8CAwgAvnNzf6Egd6ep6rkudhih7bSuDByNWj8B5yrsIuY+4gkCAQcAvOYQCKDzFpnFDCHNtlEROwa97X10qnZ5jR0xMBl8KmYCAwcAltoc7MVlI4wSZINq2ueV50tLMmoT+n+X6IyfVqBr1QACAgkAiOXC3QSin967T3d1oB1gXFpzPyI2S0LKZklKoPk4Fj0CBQYAsmpWIEcgCRoL84qDK0OKK3c29NgY+LCw/yhF7TGjsx4CBAYA+lKYyWAaNHS7bjgqnO6ffLws8mjx3MlHZZ4uiIaUf38CAgcAJo7qe2oownAuCN1tXaKgqWOyyBzJto7XC+qMRXl1NVkCBQkAWHP4KqkU0gEETtzxhmcO12VObkOl+hGQU4B4AzQ1fycCAQYA0hA1avKqvsxMV2EXDdQBIe3aqIqq4BRWS9X2ah+w3DA=",
+  "master_secret_key": "eSjfjtbIV3CzPa4qlpbN2ovYplSjSScSr8GKyuRdkA3Yi7XTdIUANaRg+qEmfHKpIeoe43dPVEt7K2Sy0k6CB9koCivg3TJ77k/8nFp0rFTI8T1aVuKnYr48RUjT6KQMFAIEBwDKPz0sARWwPANw5rsBVHVpNmQbcxa9F4XKsYFubM+IBgIECABRYFuWZOCWaoXrUNjauBxQ02KWwvk3Z23VDjzAsGXlAgIBBwBTRvC/zukGK7pGauKtjDJI/JH72ZoslCRFN3Mb/zsoDAIBCQCjfz5GUs7myLtO+AU/yOxdkRo8O13ds4EfFzXLG7ONDgIECQDLXIJ/IhxYjuonQpmGyPIA2nwYousRVznWh2VNce4cDgIBCAD3wwwsGrnYGnh1g4ylTedf9mBHplP38bj6hrwlS15dBQICBgC16XWLoB9caBlIlsDRA5p+lmn8S437r0y3t9x7+89bAwIDCABHKyPQ6RFOZQ90vVgJa47TvsXQizbZaonW/M1EXf4ODQICCACEdQlGueZ1QsHIKgja/pwqJnBYnjfAmVuF99ma7h3eDQIEBgAr6+Gk30iRIgh7pDOGmXIg8nIEpF988jReDpA77o8aCgIFBgDo3UVkzX3Lkti5NhKc6UPquVw3S55G18ZtIqPO9cyQDAIFBwAenNFbQVkI7fEtq5vEJaWXziwljzkKy59ZxPobvWLNBQIDBwASqJx9VuOdvyn5s7aKitYnxbv8/tQP5FVvTWevaNs4BQICBwCZDLLRBkftb0SZY42ZL5slwprpmwR3NC6grDnB+8L3AwIBBgD/2+TfWjsNIaoOe7e2b3daP52qnxxPnFt/HE1WyG8DCwIDBgBp8j7ppBHwzWjD6BRv6v4J0F9bHdu539cP/yapS0F5BAIDCQC+RgpTX9ymZti7BeZasntcjdZblYc/8rWAGuO7RTRdBwIFCQCn2JV2pf3aIjQ1Hd2KRtaGj7X/Sm/mR8damMHKTgYiDgICCQC5tM0yTGjEZD2zZpyEDQYiXVZS4N7qBmUYWNHaXpauBAIFCAAOxO2VSqzBWnG59UsCQ03Ic9+x1EiTvChQwBrZY38fAg==",
+  "policy": "{\"last_attribute_value\":9,\"max_attribute_creations\":100,\"axes\":{\"Security Level\":[[\"Protected\",\"Low Secret\",\"Medium Secret\",\"High Secret\",\"Top Secret\"],true],\"Department\":[[\"R&D\",\"HR\",\"MKG\",\"FIN\"],false]},\"attributes\":{\"Security Level::Top Secret\":[[5],\"Classic\"],\"Security Level::High Secret\":[[4],\"Classic\"],\"Security Level::Protected\":[[1],\"Classic\"],\"Security Level::Medium Secret\":[[3],\"Classic\"],\"Department::MKG\":[[8],\"Classic\"],\"Department::R&D\":[[6],\"Classic\"],\"Department::FIN\":[[9],\"Classic\"],\"Department::HR\":[[7],\"Classic\"],\"Security Level::Low Secret\":[[2],\"Classic\"]}}",
```

Comment:
I think that the policy in the non regression vector should remain a base64 encoded JSON string to match the other repositories.

Reply:
The policy is now supposed to be a black box outside of `abe_policy`. We may
break the JSON formatting in the future.

Maybe we need to change the policy format to bytes in the other repositories
instead ?

Reply:
yes, in this case the format must be changed in every repository.